tests/greenheart/test_hydrogen/test_h2_transport/test_h2_compressor.py

```diff
@@ -69,7 +69,7 @@ def test_system_power_report(self):
         )
         comp.compressor_power()
         _motor_rating, total_system_power = comp.compressor_system_power()
-        assert total_system_power == approx(3627.3907562149357)
+        assert total_system_power == 3627.3907562149357
```

Comment:
This smells like something unrelated to the PR? Do you know if it should be reverted?

Reply:
This could be a bad conflict resolution from rebasing this branch onto develop, thanks for flagging!